routes/financial/financial.controller.js

```diff
@@ -22,5 +22,6 @@ const postConfirmIncome = (req, res, next) => {
     return res.redirect('/offramp/financial')
   }
 
+  req.session.incomeConfirmed = true
```

Comment:
This is already set as `true` in the `api/user.json` file, so probably we should initially have it `null` or something.

Reply:
oh woops, that's because of me. I should change it back 🤦‍♀